network-libp2p/src/discovery/peer_contacts.rs

```diff
@@ -452,6 +491,69 @@ impl PeerContactBook {
             }
         }
     }
+
+    /// Returns true if an address is a secure websocket connection.
+    /// If address doesn't have the websocket protocol, it will return `false`.
+    fn is_address_secure(address: &Multiaddr) -> bool {
+        address.into_iter().any(|p| matches!(p, Protocol::Wss(_)))
+    }
+
+    /// Returns true if an address is valid for dialing.
+    /// It performs basic checks against unsupported addresses.
+    pub fn is_address_valid(&self, address: &Multiaddr) -> bool {
```

Comment:
```suggestion
    pub fn is_address_dialable(&self, address: &Multiaddr) -> bool {
```
`valid` doesn't really tell me what this function does. Perhaps `dialable` would fit?